.gitignore

```diff
@@ -6,6 +6,7 @@
 /pkg/
 /spec/reports/
 /tmp/
+/.idea
```

Comment:
Please don't include developer-specific things to project `.gitignore`. Set up your [global git ignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).
```suggestion
```